src/graphql/fields.ts

```diff
@@ -5,17 +5,29 @@ export enum LinkType {
 	Web = "Link.web",
 }
 
+/**
+ * For link fields that haven't been filled
+ */
```

Comment:
Rather than stating what something is for, we can explain what the type represents. We could use something like this:

```ts
/**
 * Represents a link field without a value.
 */
```

Stating what something is for is still useful provided it is in addition to the above description.